utils/swift-api-dump.py

```diff
@@ -161,8 +161,11 @@ def print_command(cmd, outfile=""):
 # Dump the API for the given module.
 
 
-def dump_module_api((cmd, extra_dump_args, output_dir, module, quiet,
-                     verbose)):
+def dump_module_api_star(pack):
```

Comment:
Can't decide if this is better as `dump_module_api_packed` instead.

Reply:
`star` seems a meaningful term of art for anyone experienced with Python.  It's used in the standard library `starmap`, for instance.